src/main/java/racing/Judgement.java

```diff
@@ -0,0 +1,27 @@
+package racing;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Judgement {
+
+    private final List<Car> cars;
+    private final int bestScore;
+
+    public Judgement(List<Car> cars) {
+        this.cars = cars;
+        this.bestScore = getBestScore();
+    }
+
+    private int getBestScore() {
+        cars.sort((a, b) -> b.getDistance() - a.getDistance());
+        return cars.get(0).getDistance();
```

Comment:
```suggestion
        return cars.get(FRIST_INDEX).getDistance();
```
상수화를 해 의미를 명확하게 해보는 것은 어떨까요?

Reply:
리뷰내용 반영부탁드립니다 🙇

Reply:
헉 제가 resolve 버튼을 잘못 눌렀네요. 😢 반영했습니다!!